NEWS.md

```diff
@@ -1,3 +1,56 @@
+# goldfish 1.5.0
+
+## New features
+
+* New DyNAM-i model available to model face-to-face interaction data, collected through video or RFID badges.
+* Functions to transform interaction data into DyNAM-compatible objects
+
+  ```R
+  defineGroups_interaction(interactions_data, actors, seed.randomization)
+  ```
+* Estimation functions for the joining and leaving rates of a DyNAM-i model
+
+  ```R
+  estimate(
+    dependent.events_interactions ~  1 + intercept(network_interactions, joining = 1) +
+                                  ego(actors$age,joining = 1, subType = "centered") +
+                                 ego(actors$age,joining = -1, subType = "centered"),
+    model = "DyNAMi", subModel = "rate")
+  ```
+  
+* Estimation functions for the choice part of a DyNAM-i model
+
+  ```R
+  estimate(
+    dependent.events_interactions ~ diff(actors$age,subType="averaged_sum") ,
+    model = "DyNAMi", subModel = "choice", 
+    estimationInit = list(opportunitiesList = opportunities))
+  ```
+* New closure effects for `model = "DyNAM"` with `subModel = "choice"` and `model = "REM"`.
+  Documentation is updated accordingly. 
+  
+  ```R
+  cycle(list(bilatnet, contignet))
+  clSender(list(bilatnet, contignet))
+  clReceiver(list(bilatnet, contignet))
+  mixedCycle(list(bilatnet, contignet))
+  mixedClSender(list(bilatnet, contignet))
+  mixedClReceiver(list(bilatnet, contignet))
+  ```
```

Comment:
Just a comment: usually we don't put code examples in NEWS, but I'm open to arguments that it is useful? Or is this maybe more the realm of vignettes, etc?

Reply:
I propose to keep them while we update and post the vignettes. I was not aware of this convention. I might have seen the use of examples in another package.  I'll create the issue with this task.

Reply:
I also think very short pieces of code don't hurt to show what the changes are about...